lib/stimulus_reflex/sanity_checker.rb

```diff
@@ -3,12 +3,22 @@
 class StimulusReflex::SanityChecker
   JSON_VERSION_FORMAT = /(\d+\.\d+\.\d+.*)"/
 
-  def self.check!
-    return if StimulusReflex.config.on_failed_sanity_checks == :ignore
+  class << self
+    def check!
+      return if StimulusReflex.config.on_failed_sanity_checks == :ignore
+      return if called_by_generate_command?
+
+      instance = new
+      instance.check_caching_enabled
+      instance.check_javascript_package_version
+    end
+
+    private
+
+    def called_by_generate_command?
+      caller.any? { |call| call.match("generate_command") }
```

Comment:
Maybe we could even simplify the check:
```suggestion
      ARGV.include? "stimulus_reflex:config"
```